### DISCUSSION TITLE
What data do you mostly work with on MTEX?

### DISCUSSION THREAD

**Discussion by nyyssont** [mtex-toolbox/mtex#1971] [Polls]
I'm curious as to which type of files people mostly use to work on MTEX.

Personally, I prefer ctf, since the data is all in a single file unlike crc + cpr and there are no issues with parent austenite reconstruction. At some point, h5oina files would be ipmorted in such a way that parent austenite reconstruction became rather buggy. I do not know the current situation. I would switch to h5oina altogether, if it was possible to import with an option to disregard data related to other than EBSD measurements. The datasets become bigger than necessary sometimes.

I would welcome any views on respective merits and downsides of various EBSD data types. I am not a professional programmer so it is always a stretch looking into these things.

-Tuomo

**Comment by kilir** [comment]
Hi Tuomo,
with respect to .h5oina: Could you elaborate what happens when you import .h5oina and the reconstruction becomes buggy? Preferentially with an example since this this sounds rather like a bug.

I can see the point that h5oina files contain much more than one might need for a certain task. How about adding a switches such as 'skipEDX' or 'skipImages' for example? Would that help? You can try 1e537cabac348b3388e73c4c57bcc9002fb84abc

ctf/cpr+crc vs h5oina: the former format does not record all the information to obtain the absolute orientation, so if the user does not know it, it can only be guessed and the relation between orientations and maps could be wrong.

Cheers,
Rüdiger

**Reply by nyyssont** [reply]
Thanks Rüdiger. I will give this a try.

I seem to remember some of the functions in parentGrainReconstructor simply didn't work. I think I traced the problem to the EDS data being imported as a row vector, while everything else was a column vector... but this is from memory. As I said, I will have to try again. I will let you know how it goes.
